plugins/ipam/dhcp/dhcp_test.go

```diff
@@ -97,16 +108,16 @@ const (
 	hostVethName string = "dhcp0"
 	contVethName string = "eth0"
 	pidfilePath  string = "/var/run/cni/dhcp-client.pid"
```

Comment:
I don't think we need pidfilePath as a global constant anymore either.

Reply:
Agreed.  Removed.